src/TerritorialCouncil/CandidacyManager.php

```diff
@@ -90,21 +98,12 @@ public function acceptInvitation(CandidacyInvitation $invitation, Candidacy $acc
 
         $this->updateCandidature($acceptedBy);
 
+        $this->entityManager->flush();
```

Comment:
No need of `flush`, it's called before in `updateCandidature`.

Reply:
good catch 👍